containers/nginx/dev.conf

```diff
@@ -1,3 +1,12 @@
+# Dynamic language redirection.
+# Ensure that "en" is last, or people with, for instance, "fr en" will always default to "en".
+map $http_accept_language $lang {
+    default en;
+    ~es es;
+    ~fr fr;
+    ~en en;
```

Comment:
I recommend adding at least one other language to the dev environment so proper testing can be done. Quizás español?